app/lib/send-email.js

```diff
@@ -26,13 +32,23 @@ const sendEmail = async (email, personalisation, reference, templateId) => {
   return success
 }
 
-const sendCarbonCopy = async (templateId, personalisation) => {
+const sendCarbonCopy = async (templateId, personalisation, reference) => {
   if (config.notify.carbonCopyEmailAddress) {
     await sendSFDEmail(
       templateId,
       config.notify.carbonCopyEmailAddress,
       personalisation
     )
+
+    applicationInsights.defaultClient.trackEvent({
+      name: 'email',
```

Comment:
Can we update the name of the event to be something a bit more descriptive (for here and the other place it#s raised above)? I like 'claim-email-requested'